extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -51,10 +51,10 @@ export class ValidationBubble {
     this.isVisible_ = false;
 
     /** @private @const {!Element} */
-    this.bubbleElement_ = win.document.createElement('div');
+    this.bubbleElement_ = ampdoc.win.document.createElement('div');
     this.bubbleElement_.classList.add('-amp-validation-bubble');
     this.bubbleElement_[OBJ_PROP] = this;
-    win.document.body.appendChild(this.bubbleElement_);
+    ampdoc.win.document.body.appendChild(this.bubbleElement_);
```

Comment:
Please comment why it's inserted into the global document body. I believe this is correct thing to do, but an unusual situation. So worth calling out in comments.